src/sentry/rules/conditions/event_frequency.py

```diff
@@ -63,6 +93,20 @@ def __init__(self, *args, **kwargs):
                     )
                 ],
             },
+            "comparisonType": {
+                "type": "choice",
+                "choices": list(sorted(comparison_types.items(), key=lambda item: item[1])),
+            },
+            "comparisonInterval": {
+                "type": "choice",
+                "choices": [
+                    (key, label)
+                    for key, (label, duration) in sorted(
+                        comparison_intervals.items(),
+                        key=lambda item: item[1][1],
+                    )
+                ],
+            },
```

Comment:
I think we will need to remove this from the rule as well, per my other comment I'll handle these in the fe